### PR TITLE
Further textpage improvements

### PR DESCRIPTION
**Pull request by matthijskooijman**
This PR does some more work on the textpage coding style for #135.

### CODE REVIEW

File: system/datatypes/text.php [hunk 1]
```diff
@@ -146,9 +146,13 @@ private function editAction(HyphaRequest $request) {
 				return ['redirect', $this->constructFullPath($this->pagename)];
 			}
 
+
```
Comment (laurensmartina) [inline system/datatypes/text.php]: Please remove extra new line.

File: system/datatypes/text.php [hunk 2]
```diff
@@ -146,9 +146,13 @@ private function editAction(HyphaRequest $request) {
...
 			// create form
 			$form = $this->createEditForm($request->getPostData());
 
+			// Validate
```
Comment (laurensmartina) [inline system/datatypes/text.php]: Comment are written in lower case throughout the application.

File: system/datatypes/text.php [hunk 3]
```diff
@@ -199,6 +203,11 @@ private function translateAction(HyphaRequest $request) {
 			// create form
 			$form = $this->createTranslationForm($request->getPostData());
 
+			// Validate
```
Comment (laurensmartina) [inline system/datatypes/text.php]: Comment are written in lower case throughout the application.

File: system/datatypes/text.php (hunk 2 above)
Comment (matthijskooijman) [reply system/datatypes/text.php]: Heh, never really consciously realized that, but indeed. Fixed now.

**Comment by matthijskooijman** [comment]
Thanks for the review, I have fixed your comments. Ok to merge like this?

**Comment by laurensmartina** [comment]
Trying to approve, but am unable to find the approve button.

**Comment by matthijskooijman** [comment]
Rebased and merged. Thanks for the review!

**Pull request merged**